client/components/first-view/index.jsx

```diff
@@ -100,14 +100,21 @@ const FirstView = React.createClass( {
 	},
 
 	updateDocumentStylesForVisibleFirstView() {
-		document.documentElement.classList.add( 'no-scroll', 'is-first-view-visible', 'is-first-view-active' );
+		document.documentElement.classList.add( 'no-scroll', 'is-first-view-active' );
```

Comment:
That's a great suggestion and certainly something I can take a look at cleaning up in a separate PR. I'd like to keep this scope of this one as focused on the fetching of preferences as possible.